### DISCUSSION TITLE
Add memory limits on off-query services

### DISCUSSION THREAD

**Discussion by raphael0202** [openfoodfacts/openfoodfacts-query#154] [Ideas]
Currently, there is no docker memory limits on off query services (api, postgresql).

**Comment by areebahmeddd** [comment]
what memory limits should we set for the services?

**Comment by raphael0202** [comment]
The current usage on prod:
- PostgreSQL: 25 GB
- API: 195 MB

I would suggest 1GB for API and 40 GB for PostgreSQL.
